subsys/bluetooth/audio/bap_broadcast_assistant.c

```diff
@@ -184,14 +191,159 @@ static int parse_recv_state(const void *data, uint16_t length,
 	return 0;
 }
 
+static void bap_ntf_read_done(void)
```

Comment:
Since this is (or could be) used for long reads without notification and the write operation, you could call it `bap_long_op_reset` or `bap_long_op_done`